docs/hub/hugging-face-hub.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Hugging Face Hub
+---
+
+<h1>Hugging Face Hub documentation</h1>
+
+## What's the Hugging Face Hub?
+
+We are helping the community work together towards the goal of advancing Artificial Intelligence 🔥.
+
+Not one company, even the Tech Titans, will be able to “solve AI” by themselves – the only way we'll achieve this is by sharing knowledge and resources. We are building the largest open-source collection of models, datasets, and metrics on the Hugging Face Hub to democratize and advance AI for everyone 🚀. The Hugging Face Hub works as a central place where anyone can share and explore models and datasets.
```

Comment:
Agreed, I think splitting it so that it's one paragraph about the functionality of the Hub and a second one about the philosophy behind it is useful. I took what you wrote and rephrased it just a bit, let me know what you think!

Reply:
Looks good!